skbio/metadata/tests/test_interval.py

```diff
@@ -468,6 +468,31 @@ def test_duplicate_bounds(self):
         self.assertEqual(len(intvls), 1)
         self.assertTrue(intvl is intvls[0])
 
+    def test_concat_empty(self):
+        for i in 0, 1, 2:
+            obs = IntervalMetadata.concat([self.im_empty] * i)
+            exp = IntervalMetadata(self.upper_bound * i)
+            self.assertEqual(obs, exp)
+
+    def test_concat(self):
```

Comment:
That is correct, since the actual coordinate spaces are being concatenated together.

for example,  if `len(seq1)=10` and `len(seq2)=20`, then `len(concat((seq1, seq2)))==30`.
So by default, the coordinates should not be overlapping, unless negative coordinates, or coordinates larger than the sequence length are allowed.